src/compiler/checker.ts

```diff
@@ -960,8 +964,15 @@ namespace ts {
             if (targetSymbol) {
                 const name = specifier.propertyName || specifier.name;
                 if (name.text) {

```

Comment:
what about namespace imports? we should add handling in `getTargetOfNamespaceImport`